README.md

```diff
@@ -41,7 +41,7 @@ import org.apache.arrow.datafusion.SessionContexts;
 public class DataFusionDemo {
 
     public static void main(String[] args) throws Exception {
-        try (ExecutionContext executionContext = ExecutionContexts.create()) {
+        try (SessionContext executionContext = SessionContexts.create()) {
```

Comment:
Should probably rename `executionContext` too

Reply:
will do